src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
TestCustomPipeNameCreationTooLongOnNonWindows
```powershell
Assert.True() Failure
Expected: True
Actual:   False
   at PSTests.Parallel.NamedPipeTests.TestCustomPipeNameCreationTooLongOnNonWindows() in D:\a\1\s\test\xUnit\csharp\test_NamedPipe.cs:line 46
```